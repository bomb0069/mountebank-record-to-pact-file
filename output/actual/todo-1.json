{
  "consumer": {
    "name": "Localhost"
  },
  "provider": {
    "name": "jsonplaceholder"
  },
  "interactions": [
    {
      "description": "",
      "request": {
        "method": "GET",
        "path": "/todos/1"
      },
      "response": {
        "status": 200,
        "headers": {
          "Content-Type": "application/json; charset=utf-8"
        },
        "body": {
          "completed": false,
          "id": 1,
          "title": "delectus aut autem",
          "userId": 1
        }
      }
    }
  ],
  "metadata": {
    "pactSpecification": {
      "version": "2.0.0"
    }
  }
}