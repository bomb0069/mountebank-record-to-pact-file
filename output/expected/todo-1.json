{
  "consumer": {
    "name": "Localhost"
  },
  "provider": {
    "name": "JsonPlaceholder"
  },
  "interactions": [
    {
      "description": "get first todo will return delectus aut autem",
      "request": {
        "method": "GET",
        "path": "/todos/1"
      },
      "response": {
        "status": 200,
        "headers": {
          "Content-Type": "application/json; charset=utf-8"
        },
        "body": {
          "userId": 1,
          "id": 1,
          "title": "delectus aut autem",
          "completed": false
        }
      }
    }
  ],
  "metadata": {
    "pactSpecification": {
      "version": "2.0.0"
    }
  }
}